nexus/src/app/background/tasks/support_bundle_collector.rs

```diff
@@ -728,14 +762,21 @@ async fn sha2_hash(file: &mut tokio::fs::File) -> anyhow::Result<ArtifactHash> {
     Ok(ArtifactHash(digest.as_slice().try_into()?))
 }
 
-async fn write_command_result_or_error<D: std::fmt::Debug>(
+/// Run a `sled-dianostics` future and save it's output to a corresponding file.
```

Comment:
Turbo nit:

```suggestion
/// Run a `sled-dianostics` future and save its output to a corresponding file.
```

Reply:
Fixed in [6b02fb3](https://github.com/oxidecomputer/omicron/pull/7461/commits/6b02fb3d6c70f42ed7bff6ccd8c4da560db48f6f)